src/test/java/io/jenkins/plugins/opentelemetry/backend/elastic/ElasticStack.java

```diff
@@ -60,81 +46,71 @@ public class ElasticStack extends DockerComposeContainer<ElasticStack> {
     private ElasticBackend elasticBackendConfiguration;
     private LogStorageRetriever elasticsearchRetriever;
 
-    public ElasticStack() {
-        super(new File("src/test/resources/docker-compose.yml"));
-        withExposedService(ELASTICSEARCH_SERVICE, ELASTICSEARCH_PORT)
-        .withExposedService(KIBANA_SERVICE, KIBANA_PORT)
-        .withExposedService(EDOT_SERVICE, OTEL_PORT)
-        .waitingFor(EDOT_SERVICE, new DockerHealthcheckWaitStrategy())
-        .withStartupTimeout(Duration.ofMinutes(10));
+    private static final String suffixService(String service) {
+        return service + SERVICE_SUFFIX;
     }
 
-    /**
-     * @return the RestClientBuilder to create the Elasticsearch REST client.
-     */
-    public Rest5ClientBuilder getBuilder() {
-        UsernamePasswordCredentials credentials = new UsernamePasswordCredentials(USER_NAME, PASSWORD.toCharArray());
-
-        BasicCredentialsProvider credentialsProvider = new BasicCredentialsProvider();
-        try{
-            credentialsProvider.setCredentials(new AuthScope(HttpHost.create(getEsUrl())), credentials);
-        } catch(URISyntaxException e) {
-            throw new IllegalArgumentException(e);
-        }
-        CloseableHttpAsyncClient httpclient = HttpAsyncClients.custom()
-            .setDefaultCredentialsProvider(credentialsProvider)
-            .build();
-
-        Rest5ClientBuilder builder = Rest5Client.builder(URI.create(getEsUrl()))
-            .setHttpClient(httpclient);
-        return builder;
+    public ElasticStack() {
+        super(new File("src/test/resources/docker-compose.yml"));
+        this.withExposedService(ELASTICSEARCH_SERVICE, ELASTICSEARCH_PORT)
+                .withExposedService(KIBANA_SERVICE, KIBANA_PORT)
+                .withExposedService(EDOT_SERVICE, OTEL_PORT)
+                .waitingFor(EDOT_SERVICE, new DockerHealthcheckWaitStrategy())
+                .withStartupTimeout(Duration.ofMinutes(10));
```

Comment:
avod changes on indentation, these changes force me to review things are not changed